CONTRIBUTING.md

```diff
@@ -1 +1,66 @@
-# Contributing to shortcode
+# Contributing
```

Comment:
**Note:** 
I've moved the contributing docs here to keep the readme clean, as well as added a section `# Tips on working with the package locally` to share my own notes on using cookiecutter-wagtail-package.